deploy/services-demo/conf/brig.demo.yaml

```diff
@@ -27,9 +27,10 @@ gundeck:
 aws:
   sesQueue: integration-brig-events
   internalQueue: integration-brig-events-internal
+  journalQueue: integration-user-events
```

Comment:
journaling is not needed in demo-mode, let's comment it out here.

Reply:
👍